src/common/meta/src/key/datanode_table.rs

```diff
@@ -195,7 +186,9 @@ impl DatanodeTableManager {
                         engine: engine.to_string(),
                         region_storage_path: region_storage_path.to_string(),
                         region_options: region_options.clone(),
-                        region_wal_options: filtered_region_wal_options,
+                        // FIXME(weny): Before we store all region wal options into table metadata or somewhere,
+                        // We must store all region wal options.
+                        region_wal_options: region_wal_options.clone(),
```

Comment:
Why store all the regions' WAL options here, or why remove the `filtered_region_wal_options`?

Reply:
Updating the `TableRoute` value requires retrieving all wal options from somewhere(i.e., TableInfo). Then, we can filter the unnecessary WAL options. However, now, Each `DatanodeTable` only stores the part of WAL options.